tsa/tsacmd/command.go

```diff
@@ -85,6 +85,10 @@ func (cmd *TSACommand) Runner(args []string) (ifrit.Runner, error) {
 		return nil, fmt.Errorf("failed to load team authorized keys: %s", err)
 	}
 
+	if cmd.getNumberOfWorkerSSHKeys(cmd.AuthorizedKeys, cmd.TeamAuthorizedKeys) == 0 {
```

Comment:
this could just be `len(cmd.Authorized) == 0 && len(cmd.TeamAuthorizedKeys) == 0` - don't think we need the helper function

come to think of it, should we maybe just skip this check and allow it to be started with no keys? that may be useful for e.g. debugging or temporarily disabling all keys (and still wanting to run the `web` node)